README.md

```diff
@@ -94,3 +95,22 @@ The following commands are available for the natively installed apps:
 # See logs of systemd services
 ./cli.sh logs [app]
 ```
+
+## Known Issues: 
+
+### GC Not Responding
+
+If the GC (Game Controller) is unresponsive to any event, try deleting the state file. This issue may occur if the GC has been active for an extended period and has recognized too many events.
+
+**Caution:** Deleting the state file will erase the current game state for the GC.
+
+```shell
+
+./cli.sh stop # stop apps
+
+cd ~/.config/ssl-game-controller/config
```

Comment:
you can skip the `cd` and just delete the file with `rm ~/.config/ssl-game-controller/config/state-store.json.stream`.